src/Umbraco.Web.UI.Client/src/views/packages/edit.controller.js

```diff
@@ -26,6 +26,9 @@
 
         vm.buttonLabel = "";
 
+        vm.packageVersionRegex = /^(\d+\.)(\d+\.)?(\*|\d+)$/;
```

Comment:
I think we should always required the `major.minor.patch` format, so only one regex necessary to cover both.

Reply:
Was also my first opion, but then looking at the packages I saw some using the 1.1 syntax. But will change it to using one format. Update will follow asap.

Reply:
Sure, we would like to encourage some more consistent versioning here so we know what to expect when people install package. Thanks for the update, I'll have a look at it this week!